myhpom/migrations/0018_scribble_data.json

```diff
@@ -0,0 +1,15 @@
+{
+	"delete": [
+    {"slug": "next-steps-header", "url": "shared", "content": "{% lorem 24 w %}"},
+    {"slug": "next-steps-download", "url": "shared", "content": "<p>\n{% lorem 36 w %}\n</p>"},
+    {"slug": "next-steps-upload", "url": "shared", "content": "<p>\n{% lorem 36 w %}\n</p>"}
+  ],
+	"create": [
+    {"slug": "header-text", "url": "next_steps", "content": "You are joining millions across the country in owning your own health by planning for care in advance. Your next steps:"},
```

Comment:
I'm confused about these sections that have actual legitimate looking content - if we have the language from the client, we don't need a snippet for it.

Reply:
(resolved in chat)